gunicorn/datadog_checks/gunicorn/data/conf.yaml.example

```diff
@@ -14,6 +14,11 @@ instances:
     #
   - proc_name: <WEB_APP_NAME>
 
+    ## @param gunicorn_bin_path - string - optional
+    ## gunicorn_bin_path used for retrieving metadata e.g. version
```

Comment:
Can we document how to use `docker exec` similar to https://github.com/DataDog/integrations-core/blob/master/cassandra_nodetool/datadog_checks/cassandra_nodetool/data/conf.yaml.example#L12 ?

Reply:
See other related answer below.